stable/astro/README.md

```diff
@@ -12,7 +12,8 @@ More information about astro is available on [GitHub](https://github.com/Fairwin
 We recommend installing astro in its own namespace and with a simple release name:
 
 ```
-helm install incubator/astro --name astro --namespace astro
+helm repo add fairwinds-stable https://charts.fairwinds.com/stable
+helm install fairwinds-stable/astro --name astro --namespace astro
```

Comment:
for helm 3 command should be different 
helm 2 - helm install fairwinds-stable/astro --name astro --namespace astro
helm 3 - helm install astro fairwinds-stable/astro --namespace astro

Reply:
Thanks for pointing that out! Fixed for polaris and rbac-manager too